bot/cogs/webhook_remover.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import re
+import typing as t
+
+from discord import Colour, Message
+from discord.ext.commands import Cog
+
+from bot.bot import Bot
+from bot.cogs.moderation.modlog import ModLog
+from bot.constants import Channels, Colours, Event, Icons
+
+WEBHOOK_URL_RE = re.compile(r"discordapp\.com/api/webhooks/\d+/\S+/?")
+
+ALERT_MESSAGE_TEMPLATE = (
+    "{user}, looks like you posted Discord Webhook URL to chat. "
+    "I removed this, but we **strongly** suggest to change this now "
+    "to prevent any spam abuse to channel. Please avoid doing this in future. "
+    "If you believe this was mistake, please let us know."
+)
```

Comment:
This is more in line with the wording for the token message. It also avoids the awkward mix of pronouns (_I_ and _we/us_).

```suggestion
ALERT_MESSAGE_TEMPLATE = (
    "{user}, looks like you posted a Discord webhook URL. Therefore, your "
    "message has been removed. Your webhook may have been **compromised** so "
    "please re-create the webhook **immediately**. If you believe this was "
    "mistake, please let us know."
)
```